tests/cross_functional/ui/test_validation_ui.py

```diff
@@ -173,3 +174,20 @@ def test_ocs_operator_is_not_present(self, setup_ui_class_factory):
             "OCS operator is present in the installed operator tab, expected to see only ODF "
             "operator"
         )
+
+    @ui
+    @tier1
+    @runs_on_provider
+    @pytest.mark.bugzilla("XXXX")
```

Comment:
do we want to put something instead of "XXXX" before merge?

Reply:
@pintojoy we dont have bz for this test.. correct?